src/core_plugins/kibana/public/discover/index.html

```diff
@@ -139,6 +146,13 @@ <h2>Searching</h2>
                  style="height: 200px"
                 >
               </visualization>
+
+              <div>
+                  <center class="small">
```

Comment:
`center` is a [deprecated html tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center). I'm guessing you're using it because it's already used in the vis header. Since it's only used in those two places, I think it's worth fixing. It looks like they both only contain inline elements, so you should be able to just change it to a div and add `text-align:center;`